src/plugins/data/common/search/aggs/agg_types_registry.ts

```diff
@@ -9,6 +9,7 @@
 import type { BucketAggType } from './buckets/bucket_agg_type';
 import type { MetricAggType } from './metrics/metric_agg_type';
 import type { AggTypesDependencies } from './agg_types';
+import { legacyAggs } from './legacy_aggs';
```

Comment:
I decided against making legacyAggs an instance property of AggTypesRegistry with a setup registration function and registering geohash_grid in maps plugin since BucketAggType and IBucketAggConfig are not exported from data/common.